### ISSUE TITLE
Make it available for PY-223-*

### ISSUE THREAD

**Issue by chbndrhnns** [koxudaxi/pydantic-pycharm-plugin#548]
I'd like to use the plugin in the 223* builds.

**Comment by chbndrhnns** [comment]
A couple of tests are currently failing:

```
com.koxudaxi.pydantic.PydanticCompletionTest > testClassFields FAILED
    junit.framework.AssertionFailedError at PydanticCompletionTest.kt:43

com.koxudaxi.pydantic.PydanticCompletionTest > testParameterAnnotationType FAILED
    junit.framework.AssertionFailedError at PydanticCompletionTest.kt:43

com.koxudaxi.pydantic.PydanticCompletionTest > testClass FAILED
    junit.framework.AssertionFailedError at PydanticCompletionTest.kt:43

com.koxudaxi.pydantic.PydanticCompletionTest > testClassValidatorCls FAILED
    junit.framework.AssertionFailedError at PydanticCompletionTest.kt:43

com.koxudaxi.pydantic.PydanticCompletionTest > testKeywordArgumentDotName FAILED
    java.lang.NullPointerException at PydanticCompletionTest.kt:36

com.koxudaxi.pydantic.PydanticCompletionTest > testKeywordArgumentDot FAILED
    junit.framework.AssertionFailedError at PydanticCompletionTest.kt:43

com.koxudaxi.pydantic.PydanticCompletionTest > testClassMethodCls FAILED
    junit.framework.AssertionFailedError at PydanticCompletionTest.kt:43
```

**Comment by koxudaxi** [comment]
@chbndrhnns 
Thank you for your comment.
I released the new version as [0.3.14](https://plugins.jetbrains.com/plugin/12861-pydantic/versions/stable/239547)
